engine/gfx/sprites.asm

```diff
@@ -267,9 +269,21 @@ UpdateAnimFrame:
 	ld a, [hl]
 	cp -1 ; this lets the party menu icons keep their dynamic color attribute
 	jr z, .skipOAMAttributes
+	cp -2
```

Comment:
When the only "magic" value was -1, I didn't mind it being hard-coded (plus it was a long time ago and I had just copied it that way from TPP:AC or Prism). But if -2 is also significant, I think this is a good time to introduce named constants.

Reply:
This is needed for the new `FLY_MON` sprite animation. We needed the sprite's selected Obj palette slot to be dynamic `-1`, but the overworld sprite we also needed to preserve `X_FLIP`, so we use `-2`. I'll come up with good constants for these.